pkg/client/templatebrokeruuidmaps.go

```diff
@@ -0,0 +1,82 @@
+)
+
+// TemplateBrokerUUIDMapsInterfacer has methods to work with TemplateBrokerUUIDMap resources
+type TemplateBrokerUUIDMapsInterfacer interface {
+	TemplateBrokerUUIDMaps() TemplateBrokerUUIDMapInterface
+}
+
+// TemplateBrokerUUIDMapInterface exposes methods on TemplateBrokerUUIDMap resources.
+type TemplateBrokerUUIDMapInterface interface {
+	List(opts kapi.ListOptions) (*templateapi.TemplateBrokerUUIDMapList, error)
+	Get(name string) (*templateapi.TemplateBrokerUUIDMap, error)
+	Create(templatebrokeruuidmap *templateapi.TemplateBrokerUUIDMap) (*templateapi.TemplateBrokerUUIDMap, error)
+	Update(templatebrokeruuidmap *templateapi.TemplateBrokerUUIDMap) (*templateapi.TemplateBrokerUUIDMap, error)
+	Delete(name string, opts *kapi.DeleteOptions) error
+	Watch(opts kapi.ListOptions) (watch.Interface, error)
+}
+
+// templatebrokeruuidmaps implements TemplateBrokerUUIDMapsNamespacer interface
+type templatebrokeruuidmaps struct {
+	r *Client
+}
+
+// newTemplates returns a templatebrokeruuidmaps
+func newTemplateBrokerUUIDMaps(c *Client) *templatebrokeruuidmaps {
+	return &templatebrokeruuidmaps{
+		r: c,
+	}
+}
+
+// List returns a list of templatebrokeruuidmaps that match the label and field selectors.
+func (c *templatebrokeruuidmaps) List(opts kapi.ListOptions) (result *templateapi.TemplateBrokerUUIDMapList, err error) {
+	result = &templateapi.TemplateBrokerUUIDMapList{}
+	err = c.r.Get().
+		Resource("templatebrokeruuidmaps").
+		VersionedParams(&opts, kapi.ParameterCodec).
+		Do().
+		Into(result)
+	return
+}
+
+// Get returns information about a particular templatebrokeruuidmap and error if one occurs.
+func (c *templatebrokeruuidmaps) Get(name string) (result *templateapi.TemplateBrokerUUIDMap, err error) {
+	result = &templateapi.TemplateBrokerUUIDMap{}
+	err = c.r.Get().Resource("templatebrokeruuidmaps").Name(name).Do().Into(result)
+	return
+}
+
+// Create creates new templatebrokeruuidmap. Returns the server's representation of the templatebrokeruuidmap and error if one occurs.
+func (c *templatebrokeruuidmaps) Create(templatebrokeruuidmap *templateapi.TemplateBrokerUUIDMap) (result *templateapi.TemplateBrokerUUIDMap, err error) {
+	result = &templateapi.TemplateBrokerUUIDMap{}
+	err = c.r.Post().Resource("templatebrokeruuidmaps").Body(templatebrokeruuidmap).Do().Into(result)
+	return
+}
+
+// Update updates the templatebrokeruuidmap on server. Returns the server's representation of the templatebrokeruuidmap and error if one occurs.
+func (c *templatebrokeruuidmaps) Update(templatebrokeruuidmap *templateapi.TemplateBrokerUUIDMap) (result *templateapi.TemplateBrokerUUIDMap, err error) {
+	result = &templateapi.TemplateBrokerUUIDMap{}
+	err = c.r.Put().Resource("templatebrokeruuidmaps").Name(templatebrokeruuidmap.Name).Body(templatebrokeruuidmap).Do().Into(result)
+	return
+}
+
+// Delete deletes a templatebrokeruuidmap, returns error if one occurs.
+// QUERY: the opts *kapi.DeleteOptions here seems non-standard, is that OK?
```

Comment:
the addition of opts *kapi.DeleteOptions to the arguments - didn't think that seemed to appear elsewhere in pkg/client